types/har-format/index.d.ts

```diff
@@ -634,24 +635,48 @@ export interface QueryString {
  *
  * http://www.softwareishard.com/blog/har-12-spec/#postData
  */
-export interface PostData {
+export type PostData = PostDataCommon & (PostDataParams | PostDataText);
+
+/**
+ * The common properties of PostData
+ */
+export interface PostDataCommon {
     /** Mime type of posted data. */
     mimeType: string;
+    /**  A comment provided by the user or the application */
+    comment?: string;
+}
+
+/**
+ * Post data with `params` specified.
+ */
+export interface PostDataParams {
     /**
      * List of posted parameters (in case of URL encoded parameters).
-     *
-     * _`text` and `params` fields are mutually exclusive._
      */
     params: Param[];
+
     /**
-     * Plain text posted data
-     *
      * _`params` and `text` fields are mutually exclusive._
      */
+    text?: never;
```

Comment:
Following the comment _`params` and `text` fields are mutually exclusiv_ would it make sense to enforce this? :

```typescript
export type PostData = PostDataCommon & ({ params: Param[] } | { text: string });
```